icons/turtle.svg

```diff
@@ -0,0 +1,16 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="m12 10 2 4v3a1 1 0 0 0 1 1h2a1 1 0 0 0 1-1v-3a8 8 0 0 0-16 0v3a1 1 0 0 0 1 1h2a1 1 0 0 0 1-1v-3l2-4h4Z" />
+  <path d="M4.824 7.9 8 10" />
+  <path d="M15.176 7.9 12 10" />
+  <path d="M16.9 10H20a2 2 0 0 1 0 4H2" />
```

Comment:
Still not quite right (although mea culpa, the previous one I submitted was also wrong 🤦‍♂️):
```suggestion
  <path d="m12 10 2 4v3a1 1 0 0 0 1 1h2a1 1 0 0 0 1-1v-3a8 8 0 1 0-16 0v3a1 1 0 0 0 1 1h2a1 1 0 0 0 1-1v-3l2-4h4Z" />
  <path d="M8 10 4.81 7.9" />
  <path d="m12 10 3.19-2.1" />
  <path d="M16.93 10H20a2 2 0 0 1 0 4H2" />
```